t/t1005-max-jobs-limits.t

```diff
@@ -161,8 +161,8 @@ test_expect_success 'make sure jobs are still running' '
 '
 
 test_expect_success 'cancel all remaining jobs' '
-	flux job cancel ${jobid1} &&
-	flux job cancel ${jobid2}
+	flux cancel ${jobid1} &&
+	flux cancel ${jobid2}
```

Comment:
This is a _**super**_ nit but I noticed in flux we generally do the `${some_variable}` convention in shell scripts/sharness instead of `$some_variable`. Highlighting this one since it's different from the others (but do not feel the need to change it, that's way too pedantic.)

Reply:
Good catch, thanks! I just force-pushed up a fix to go through the testsuite and wrap the variables that didn't have brackets with brackets. Hope it's okay that I tag it along in this PR